test/smoke/src/helpers/setupEnvironmentHelper.ts

```diff
@@ -296,6 +296,13 @@ import * as Icon from '@expo/vector-icons';
         const metroConfigPath = path.join(appPath, "metro.config.js");
         console.log(`*** Patching  ${metroConfigPath}`);
         const patchContent = `
+// To avoid EPERM errors on Windows
```

Comment:
Do we have an URL link for this change describing why this change is needed?

Reply:
Unfortunately, we haven't one. I'll add more details though.